feature/bookmarks/src/test/kotlin/com/google/samples/apps/nowinandroid/feature/bookmarks/BookmarksViewModelTest.kt

```diff
@@ -88,6 +90,54 @@ class BookmarksViewModelTest {
         val item = viewModel.feedUiState.value
         assertIs<Success>(item)
         assertEquals(item.feed.size, 0)
+        assertTrue(viewModel.shouldDisplayUndoBookmark)
+
+        collectJob.cancel()
+    }
+
+    @Test
+    fun whenResourceViewed_setResourcesViewed() = runTest {
```

Comment:
According this [wiki](https://github.com/android/nowinandroid/wiki/Testing-strategy-and-how-to-test#naming) you shouldn't use `when` and `then` in the test name.
> Other rules for consistency:
>
> - Do not add the subject under test in test names, it should be already present in the class name.
> - Do not add "given", "when" or "then".
> - Use lowerCamelCase separated by underscores.

Reply:
@Jaehwa-Noh 
Thank you for your review!
I didn't realize the wiki existed. I update test name.